config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1499,3 +1499,87 @@ presubmits:
           requests:
             cpu: 4
             memory: 6Gi
+  - name: pull-kubernetes-node-swap-ubuntu-serial
+    cluster: k8s-infra-prow-build
+    always_run: false
+    optional: true
+    skip_report: false
+    annotations:
+      testgrid-dashboards: sig-node-presubmits
+      testgrid-tab-name: pr-kubelet-gce-e2e-swap-ubuntu-serial
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220117-99a00602a5-master
+          args:
+            - --root=/go/src
+            - "--job=$(JOB_NAME)"
+            - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+            - "--service-account=/etc/service-account/service-account.json"
+            - "--upload=gs://kubernetes-jenkins/pr-logs"
+            - "--timeout=240"
+            - --scenario=kubernetes_e2e
+            - --
+            - --deployment=node
+            - --gcp-zone=us-west1-b
+            - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/swap/image-config-swap.yaml
+            - --node-test-args=--container-runtime=remote --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --feature-gates=NodeSwap=true --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --fail-swap-on=false --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd*\"]}"
+            - --node-tests=true
+            - --provider=gce
+            - --test_args=--nodes=8 --focus="\[Serial\]" --skip="\[Flaky\]|\[Slow\]"
```

Comment:
These are serial tests so we shouldn't have 8 nodes. Args should match https://github.com/kubernetes/test-infra/blob/cc95789ad64e7e0234b418781d4f01cac31928cd/config/jobs/kubernetes/sig-node/node-kubelet.yaml#L11-L24

but we should use the swap image config and have the swap feature flag enabled.

Reply:
I changed the job following your example. 

The only question is that why we use `/workspace/test-infra/jobs/e2e_node/containerd/image-config-serial.yaml` not `image-config-swap-fedora.yaml` for Swap serial tests.